### DISCUSSION TITLE
Why are there two URLs for ASF Boxer Service?

### DISCUSSION THREAD

**Discussion by linghengqian** [apache/infrastructure-ideas#7] [Q&A] [answered]
- I noticed that there are two URLs for ASF Boxer Service, https://gitbox.apache.org/boxer/ and https://git-wip-us.apache.org/boxer/ . The two URLs are independent of each other and require repeated OAuth login. This seems a bit redundant. Why is this done?
- ![image](https://github.com/user-attachments/assets/d5d9fd77-822e-4ddd-8070-ec3d4edeea0a)
- ![image](https://github.com/user-attachments/assets/65086e91-79fe-4b4c-9177-dd5f9c08e0d4)

**Comment by Humbedooh** [comment, marked as answer]
They are the same host. For historical compatibility, we have retained the git-wip-us hostname even though we do advocate the old hostname any longer.
